src/app/config/actionContants.ts

```diff
@@ -0,0 +1,7 @@
+export const actions = {
+    PERIODS : {
```

Comment:
change the name to INTERVALS or something like that

Reply:
done